clustered/integration-test/build.gradle

```diff
@@ -59,6 +59,7 @@ task copyServerLibs(type: Copy) {
 }
 
 test {
+  jvmArgs '-Xmx512m'
```

Comment:
Could we use `maxHeapSize '512m'` instead? Also could we set `maxParallelForks 8` to cap the total heap usage at 4GB?

Reply:
While you're at it could we lower the `maxParallelForks` in the top level `build.gradle` to 16 to give the same cap?

Reply:
done.